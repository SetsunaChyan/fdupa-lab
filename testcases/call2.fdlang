function main(){
    a = input();
    b = 20;
    call callee(a,b);
}

function callee(m,n){
    m = m - 1;
    if(m >= 20){
        n = m;
    }else{
        n = n - 1;
    }
    check_interval(m, 0, 254);
    check_interval(n, 19, 19);
}