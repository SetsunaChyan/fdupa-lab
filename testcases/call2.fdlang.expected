Line 14: YES; m in [0, 254]
Line 15: NO; n in [19, 19] U [20, 254]